src/Typesense/Errors/ImportError.ts

```diff
@@ -1,10 +1,9 @@
-'use strict'
-
 import TypesenseError from './TypesenseError'
 
 export default class ImportError extends TypesenseError {
-  constructor (message, importResults) {
-    super(message)
+  importResults: any
```

Comment:
wondering if we have a stronger type here

Reply:
The type for this is `ImportResponseFail` in `Documents.ts`